internal/commands/server/k8s_e2e_test.go

```diff
@@ -245,7 +250,8 @@ func TestHTTPMeshService(t *testing.T) {
 			require.NoError(t, err)
 			serviceTwo, err := e2e.DeployHTTPMeshService(ctx, cfg)
 			require.NoError(t, err)
-			serviceThree, err := e2e.DeployHTTPMeshService(ctx, cfg)
+			// register this service in a different consul namespace
+			serviceThree, err := e2e.DeployHTTPMeshService(ctx, cfg, e2e.ConsulNamespace(ctx))
```

Comment:
this makes it so we're actually leveraging routing between multiple namespaces from a single gateway in our tests